### PR TITLE
Fix for EnableSource always true

### PR DESCRIPTION
**Pull request by jjjoness**
Signed-off-by: John Jones-Steele <82226755+jjjoness@users.noreply.github.com>

## What does this PR do?

Fixes bug #11542 

## How was this PR tested?

Checked with testbed and works correctly.

### CODE REVIEW

**Review by amzn-tmryan — APPROVED** [review]
LGTM

File: Code/Tools/AssetProcessor/native/utilities/GUIApplicationManager.cpp [hunk 1]
```diff
@@ -805,7 +805,8 @@ ApplicationManager::RegistryCheckInstructions GUIApplicationManager::PopupRegist
 
 void GUIApplicationManager::InitSourceControl()
 {
-    QSettings settings;
+    // Look in the editor's settings for the Source Control value
+    QSettings settings(QApplication::organizationName(), QString("O3DE Editor"));
     settings.beginGroup("Settings");
     bool enableSourceControl = settings.value("EnableSourceControl", 1).toBool();
```
Comment (gadams3) [inline Code/Tools/AssetProcessor/native/utilities/GUIApplicationManager.cpp:811]: Just wondering if the AP has its own settings independent of the editor

**Review by amzn-mike — APPROVED** [review]

File: Code/Tools/AssetProcessor/native/utilities/GUIApplicationManager.cpp (hunk 1 above)
Comment (amzn-mike) [reply Code/Tools/AssetProcessor/native/utilities/GUIApplicationManager.cpp:811]: In general it does, but for this it makes sense to share the setting

Comment (jjjoness) [reply Code/Tools/AssetProcessor/native/utilities/GUIApplicationManager.cpp:811]: Yes, it does but it makes sense to be the same as the Editor so in this case we use that.

**Pull request merged**